### PR TITLE
BAVL-24: Add cypress tests

### PR DESCRIPTION
**Pull request by smcveigh941**
Cypress tests for:

- Create a court booking happy path
- Create a probation booking happy path
- Create a court booking with an alternative time suggested
- Create a probation booking with an alternative time suggested
- Set and update user court preferences 
- Set and update user probation team preferences

More tests to come for view, amend and cancel

### CODE REVIEW

File: integration_tests/e2e/selectUserPreferences.cy.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      selectCourtPreferencesPage.confirm().click()
+
+      cy.task('stubGetUserCourtPreferences')
+      const courtPreferencesConfirmationPage = Page.verifyOnPage(CourtPreferencesConfirmationPage)
+      courtPreferencesConfirmationPage.continue().click()
+
+      Page.verifyOnPage(HomePage)
+    })
+
+    it('Court user can update court preferences', () => {
```
Comment (matt-crowson-moj) [inline integration_tests/e2e/selectUserPreferences.cy.ts:39]: I might be misreading but what is this updating?

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

File: integration_tests/e2e/selectUserPreferences.cy.ts (hunk 1 above)
Comment (matt-crowson-moj) [reply integration_tests/e2e/selectUserPreferences.cy.ts:39]: I get that but it doesn't update anything?

**Review by timharrison-moj — APPROVED** [review]
Excellent. LGTM.

**Pull request merged**